app/js/lib/foxbox/services/ip-camera.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+import BaseService from './base';
+
+const TYPE = 'ip-camera';
+
+export default class IpCameraService extends BaseService {
+  constructor(props, config) {
+    super(props, config);
+    Object.seal(this);
+  }
+
+  // Public getter only properties.
+  get type() {
+    return TYPE;
+  }
+
+  // Public methods.
+  getImageList() {
+    return this.get('image_list');
```

Comment:
__funny fact:__ now when camera class is small and clear it's obvious that names of channel kind for image list and latest images look inconsistent :)